DVSRegister/Views/Register/ProviderDetails.cshtml

```diff
@@ -38,7 +38,14 @@
                             Public telephone number
                         </dt>
                         <dd class="govuk-summary-list__value">
-                            @Model.Provider.ProviderTelephoneNumber
+                            @if (Model.Provider.ProviderTelephoneNumber != null && Model.Provider.ProviderTelephoneNumber.Any())
```

Comment:
for string we usually do like  if (!string.IsNullOrEmpty(Model.Provider.ProviderTelephoneNumber))